internal/provider/resource_dns_forwarder.go

```diff
@@ -300,6 +349,19 @@ func setDnsForwarderAttributes(d *schema.ResourceData, dnsForwarder dns.Networki
 		}
 	}
 
+	if dnsForwarder.Spec.Config.NetworkingV1ForwardViaGcpDnsZones != nil {
+		domainMapping := dnsForwarder.Spec.Config.NetworkingV1ForwardViaGcpDnsZones.GetDomainMappings()
+		stringMap := make(map[string]string)
+		for key, value := range domainMapping {
+			stringMap[key] = *value.Zone + "," + *value.Project
```

Comment:
To make it more robust, consider something like below, you can have better naming:

```suggestion
zone, zoneOk := value.GetZoneOk()
project, projectOk := value.GetProjectOk()
if zoneOk && projectOk {
	stringMap[key] = *zone + "," + *project
}
```